packages/safe-tools-client/app/components/deposit-modal/index.css

```diff
@@ -12,23 +12,49 @@
   font-size: var(--boxel-font-size-sm);
 }
 
+.deposit-modal__section-funds-info,
+.deposit-modal__section-funds-info-balance,
+.deposit-modal__section-funds-info-token {
+  display: flex;
+  flex-direction: row;
+}
+
 .deposit-modal__section-funds-info {
   padding: var(--boxel-sp);
   border-radius: var(--boxel-border-radius);
   background-color: var(--boxel-yellow);
-  display: flex;
-  flex-direction: row;
 }
 
 .deposit-modal__section-funds-info-header {
   font-weight: bold;
   flex: 1;
 }
 
+.deposit-modal__section p.deposit-modal__section-funds-info-balance {
+  margin: var(--boxel-sp-xxs) 0;
+}
+
 .deposit-modal__section-funds-info-icon {
   --icon-color: var(--boxel-dark);
 
   flex: 0.1;
   width: 24px;
   height: 24px;
 }
+
+.deposit-modal__section-funds-info-token {
+  font-size: 14px;
```

Comment:
In cases where the design specifies something that isn’t covered by the `--boxel-font-size…` properties, I tend toward using one of those vs specifying it numerically, just so we have more consistent appearance in the applications. `--boxel-font-size-sm` is 13px which seems close enough to me 😆